libs/application/ui-fields/src/lib/AsyncSelectFormField.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC, useEffect, useState } from 'react'
+import {
+  AsyncSelectField,
+  FieldBaseProps,
+  formatText,
+} from '@island.is/application/core'
+import { Box } from '@island.is/island-ui/core'
+import {
+  SelectController,
+  FieldDescription,
+} from '@island.is/shared/form-fields'
+import { useLocale } from '@island.is/localization'
+import { useApolloClient } from '@apollo/client/react'
+import { Option } from '@island.is/application/core'
+
+interface Props extends FieldBaseProps {
+  field: AsyncSelectField
+}
+const AsyncSelectFormField: FC<Props> = ({ application, error, field }) => {
```

Comment:
don't we need to know the loading state or the error from the graphql call?

Reply:
I don't think we need the loading state, options will just be a empty array until it is filled, but I'm adding the error state